web/usuaris/src/helpers/fetchers.ts

```diff
@@ -8,3 +8,41 @@ export const getUsers = async () => {
   const data = await response.json()
   return data
 }
+
+export const loginUserFetcher = async (user: object) => {
+  const errorMessage: { [key: number]: string } = {
+    401: 'Credenciales incorrectas.',
+    403: 'Usuario en proceso de activación. Por favor, póngase en contacto con el administrador.',
+  }
+
+  const response = await fetch(urls.logIn, {
+    method: 'POST',
+    body: JSON.stringify(user),
+    headers: { 'Content-Type': 'application/json' },
+  })
+
+  if (
+    !response.ok &&
+    Object.hasOwnProperty.call(errorMessage, response.status)
+  ) {
+    throw new Error(errorMessage[response.status])
+  }
+
+  return response.status === 204 ? null : response.json()
+}
+
+export const userInfoFetcher = async (token: string) => {
+  const response = await fetch(urls.getMe, {
+    method: 'POST',
+    body: token,
```

Comment:
Check this (required request body and stringification), please

Reply:
Is it necessary the stringification? The token is already typed as string, and it has to be in the parameter as such... Also, if it is typed as string only, means that it is required, isn't it? Maybe I am completely wrong, I would appreciate some guidance on this if so...